packages/suite/src/components/suite/FormattedNftAmount.tsx

```diff
@@ -1,89 +1,127 @@
-    color: ${({ theme }) => theme.legacy.TYPE_GREEN};
-    text-decoration: underline;
-    display: flex;
-    font-size: ${variables.FONT_SIZE.TINY};
-    line-height: initial;
-    align-items: center;
-`;
-
-const NoLink = styled.div`
-    display: flex;
-`;
-
-const Id = styled.div`
-    white-space: nowrap;
-    overflow: hidden;
-    text-overflow: ellipsis;
-    max-width: 145px;
-`;
+export const Id = ({ id }: { id?: string }) => {
+    return id ? (
+        <Text maxWidth="145px" ellipsisLineCount={1} textWrap="nowrap">
+            {id}
+        </Text>
+    ) : null;
+};
 
 export interface FormattedNftAmountProps {
     transfer: TokenTransfer;
     signValue?: SignValue;
     className?: string;
     isWithLink?: boolean;
+    alignMultitoken?: 'flex-end' | 'flex-start';
 }
 
 export const FormattedNftAmount = ({
     transfer,
     signValue,
     className,
     isWithLink,
+    alignMultitoken = 'flex-end',
 }: FormattedNftAmountProps) => {
     const id = getNftTokenId(transfer);
-
+    const theme = useTheme();
+    const { translationString } = useTranslation();
     const { selectedAccount } = useSelector(state => state.wallet);
     const { network } = selectedAccount;
     const explorerUrl =
         network?.networkType === 'ethereum'
             ? `${network?.explorer.nft}/${transfer.contract}/${id}`
             : undefined;
 
-    const idComponent = <Id>{id}</Id>;
-    const symbolComponent = transfer.symbol ? <Symbol>&nbsp;{transfer.symbol}</Symbol> : null;
+    const symbolComponent = transfer.symbol ? (
+        <Text maxWidth="120px" ellipsisLineCount={1} textWrap="nowrap">
+            {transfer.symbol}
+        </Text>
+    ) : null;
+
+    const isMultitoken = isNftMultitokenTransfer(transfer);
+
+    if (isMultitoken) {
+        const tokens = transfer.multiTokenValues;
+
+        return (
+            <HiddenPlaceholder>
+                <Column alignItems={alignMultitoken}>
+                    {tokens?.map((token, index) => (
+                        <Row key={`${token.id}-${index}`} gap={4}>
+                            <Row>
+                                {signValue ? <Sign value={signValue} /> : null}
+                                {transfer.name ? (
+                                    <BlurUrls
+                                        text={translationString('TR_COLLECTION_NAME_OF_TOKEN_ID', {
+                                            tokenValue: token.value,
+                                            collectionName: transfer.name,
+                                        })}
+                                    />
+                                ) : (
+                                    <>
+                                        {token.value}x&nbsp;
+                                        <Translation id="TR_TOKEN_ID" />:
+                                    </>
+                                )}
+                            </Row>
+                            <Row>
```

Comment:
Do you need Row here? There is only one element

Reply:
Removed